grails-app/services/io/xh/toolbox/MonitorDefinitionService.groovy

```diff
@@ -5,25 +5,103 @@ import io.xh.hoist.monitor.MonitorResult
 
 import static io.xh.hoist.monitor.MonitorStatus.OK
 import static io.xh.hoist.monitor.MonitorStatus.FAIL
+import static io.xh.hoist.monitor.MonitorStatus.UNKNOWN
+import static io.xh.hoist.monitor.MonitorStatus.WARN
 import static io.xh.hoist.util.DateTimeUtils.MINUTES
 import static java.lang.System.currentTimeMillis
 
 class MonitorDefinitionService extends BaseService {
 
     def newsService
+    def fileManagerService
+    def recallsService
 
+    /**
+     * Check the count of news stories loaded by NewsService
+     * @param result
```

Comment:
Leave the boiler plate (and unfilled in) @param and @return out here.  These will be the same for every monitor.

Looking for minimal boiler plate in our examples.